.buildkite/pipeline.yml

```diff
@@ -20,11 +24,18 @@ steps:
  - label: build library
    if: *not_scheduled
    command: nix build -L -f. lib
+ - label: LIGO-contract
+   if: build.source != "schedule"
```

Comment:
Why not `   if: *not_scheduled`?

Reply:
Also are we sure it's not needed for scheduled builds?

Reply:
hmm, this actually looks like a rebase artifact